client/state/wordads/status/selectors.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+
+import get from 'lodash/get';
+
+export function siteWordadsUnsafe( state, siteId ) {
```

Comment:
Maybe `isSiteWordadsUnsafe`. The current name suggests that this modifies something.